### ISSUE TITLE
Windows 10 5.0.1 install gets stuck at "pkgs

### ISSUE THREAD

**Issue by markwhat1** [ContinuumIO/anaconda-issues#7587]

**Comment by nehaljwani** [comment]
@markwhat1 Do you have any antivirus software installed on the machine?

**Comment by markwhat1** [comment]
Only Windows Defender, would that hold it up?

**Comment by nehaljwani** [comment]
This is the first time I am hearing about this. Did you install it for all users or just for you?

**Comment by markwhat1** [comment]
Well maybe I'm not the only one. I have tried both but my intention is to install for all users.

**Comment by nehaljwani** [comment]
Okay. Do you see similar behaviour with the Miniconda installer too?

**Comment by markwhat1** [comment]
I haven't tried to install that but I'll give it a shot and report back.

**Comment by markwhat1** [comment]
The Miniconda installer ran straight through to completion which is a bit of good news for me at least. Can I install anaconda into this python environment now and replicate the functionality of the other installer?

**Comment by nehaljwani** [comment]
Yes, you can just do:
```
conda install anaconda=5.0.1
```

**Comment by markwhat1** [comment]
Awesome, I've got that installing then. Thanks so much for the quick help, are you concerned with the problem I was having with the Anaconda3 installer? If I can help at all with that I'll certainly do my best to do so.

**Comment by nehaljwani** [comment]
Well, the logic for creating both the installers is same. It's just that the Anaconda installer has more packages. I wonder if it could be stuck for a particular package. Did you have enough free space for the installation? Were you installing it on a shared drive? How long did you want for the process to finish?

**Comment by markwhat1** [comment]
Yes sir, plenty of space and installing it on my main system drive. I waited several hours yesterday when I first encountered the problem before giving up on it when I saw that Task Manager had the pythonw.exe process listed as "Suspended". I'm currently waiting for anaconda to install into the Miniconda installation, it hasn't shown any updates since downloading the final anaconda-5.0.1 package. What's a reasonable time frame to allow for it to complete the install?

**Comment by nehaljwani** [comment]
After installation it will extract the packages and try to hard link them into the current environment. It should finish within 15 mins, after downloading all the packages.  Are you seeing this behaviour only with the 5.0* installers or you never faced this with the older ones?

**Comment by markwhat1** [comment]
The installation using Miniconda appears to have stalled as well as no progress has been made in the last hour and a half. I tried installing with the 4.4.0 installer earlier this morning with the same results.

**Comment by nehaljwani** [comment]
Could you try installing the package in the anaconda meta package one by one and see on which package it gets stuck? `conda info anaconda` should give you the list of packages.

**Comment by AmyLarson11** [comment]
I am also having the same issue where the installer is getting stuck at the install packages stage.  I am installing on a Windows 7 machine.

**Comment by moorepants** [comment]
The latest miniconda3 install worked for me. It didn't hang on the install.py command.

**Comment by seanmac10** [comment]
Same issue.

**Comment by mtowery** [comment]
After installing latest miniconda3, It didn't hang on the install.py command. 
Instead of: 
  --- old version
conda install anaconda=5.0.1 
You should do:
  --- latest version
conda update -n base conda

**Comment by xiaobabyh** [comment]
hello,everyone,I had the same problem and I solved it finally,If you can try to end up all of your antivirus software.My problem was the Acronis antivirus software.it may works finally.

**Comment by luisbonifacio** [comment]
I also had this issue here, and found the same as @xiaobabyh . I deactivated the antivirus everything just worked!
